tests/test_other.py

```diff
@@ -134,6 +136,23 @@ def do_other_test(self, dirname, emcc_args=[], run_args=[]):
     seen = run_js('a.out.js', args=run_args, stderr=PIPE, full_output=True) + '\n'
     self.assertContained(expected, seen)
 
+  def run_on_pty(self, args):
```

Comment:
how about `args => cmd`, since it's the full command, and not just some arguments?